charts/victoria-metrics-agent/templates/server.yaml

```diff
@@ -1,113 +1,124 @@
-{{- if .Values.statefulSet.enabled }}
```

Comment:
This is sadly another breaking change within a Minor upgrade :(
@AndrewChubatiuk